html/ui/js/brs.util.js

```diff
@@ -618,7 +618,7 @@ var BRS = (function(BRS, $, undefined) {
                 }
                 res += " " + hours + ":" + minutes + ":" + seconds;
 
-                if (!BRS.settings["24_hour_format"]) {
+                if (BRS.settings["24_hour_format"] == 0) {
```

Comment:
Typically you should never use double equals in JS. It was probably fine the way it was tbh.

Reply:
This was actually the fix for it. An identical as opposed to an equals would also work. I suppose therefore it's not falsy but I'm not too acquainted with javascript.